guides/asynchronous-tasks/readme.md

```diff
@@ -222,15 +224,15 @@ Async do
 end
 ```
 
-If you're letting individual tasks held by a barrier raise unhandled exceptions, be sure to call ({ruby Async::Barrier#stop}) to stop the remaining tasks:
+Unless your tasks all rescue and suppresses `StandardError`-derived exceptions, be sure to call ({ruby Async::Barrier#stop}) to stop the remaining tasks:
```

Comment:
I think you can also suppress any exceptions and it's okay, even if it's questionable to do so.